simulate_presence_script.py

```diff
@@ -0,0 +1,26 @@
+#!/usr/bin/python3
+
+import requests
+import json
+import random
+seat_seat = 'http://127.0.0.1:8000/api/seats/'
+halls_url = 'http://127.0.0.1:8000/api/halls/'
+payload = {}
+headers = {'Authorization': 'Bearer MWZiYmM5YjE3MWEwNzM0MzlmMjQwYjI0ZDE5MjMzZDMxZDA3YzcyOA=='}
+
+response_seats = requests.request("GET", seat_seat, headers=headers, data=payload)
+response_halls = requests.request('GET', halls_url, headers=headers, data=payload)
+halls = json.loads(response_halls.content)['results']
+seats = json.loads(response_seats.content)['results']
+random_nr_of_seats = random.randint(1, len(seats))
```

Comment:
"The script will first retrieve the list of Halls.
It will then retrieve the list of Seats (ids) from one of the Halls.
It will choose a random number of seats, between 1 and the number of seats."

Maybe this wasn't clear enough; sorry!
The script should select a random hall from the list of halls and then request the seats of that hall only, by its id, not all the seat objects. It's good that you added that view but you could also add another one that also applies a hall filter on them, to be used here.

We don't need to manually convert the content of the response to json.